certified-connectors/Business Assist/apiDefinition.swagger.json

```diff
@@ -0,0 +1,688 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Business Assist",
+    "description": "Business Assist APIs are using Machine Learning models which will help in improving the customer experience and provide customer success. These APIs will help you optimize your customer support that can help you forecast call volumes, analyze customer feedback, and provide self-help support for faster service. You can integrate with any platform they already use, allowing them to create their own technology stack based on both preference and value.",
```

Comment:
@sjagaleti please remove the word API from the description. It should meet the [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)